### PR TITLE
fix: update HISTORY.md

### PR DESCRIPTION
**Pull request by yotamberk**
I've updated HISTORY.md for the upcoming V6.0.0! I'm planning on officially publishing the version on October 21st . Hopefully by then all that is needed to be done for the publication can get done. 
Seems like an achievable task. 

Once we get this version published, I will archive `timeline-plus` once and for all~!

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -2,12 +2,22 @@
...
 
-## 2019-07-19, version 6.0.0
+## 2019-10-21, version 6.0.0
 
 In version 6.0.0, [timeline-plus](https://github.com/yotamberk/timeline-plus) features and PRs were ported into this library.
+We are now connected to circle-ci for future version publishes.
 
 ### New Features
 
+- feat: add an option to make the custom marker editable ([vis-timeline#117](https://github.com/visjs/vis-timeline/pull/117)) - thanks to @ryamaguchi0220
```
Comment (mojoaxel) [inline HISTORY.md]: for issues within the same project it is enough to write #117 without a link or anything.

File: HISTORY.md [hunk 2]
```diff
@@ -2,12 +2,22 @@
 
 http://visjs.org
 
-## 2019-07-19, version 6.0.0
+## 2019-10-21, version 6.0.0
```
Comment (mojoaxel) [inline HISTORY.md]: please make sure to update this before making the actual release!

File: HISTORY.md [hunk 3]
```diff
@@ -2,12 +2,22 @@
 
 http://visjs.org
```
Comment (mojoaxel) [inline HISTORY.md]: Maybe we should add something like this here:
`For release notes > v6 please visit [vis-timeline/releases at github](https://github.com/visjs/vis-timeline/releases).`

File: HISTORY.md (hunk 2 above)
Comment (yotamberk) [reply HISTORY.md]: Of course

File: HISTORY.md [hunk 4]
```diff
@@ -2,12 +2,24 @@
...
+## 2019-10-21, version 6.0.0
 
 In version 6.0.0, [timeline-plus](https://github.com/yotamberk/timeline-plus) features and PRs were ported into this library.
+We are now connected to circle-ci for future version publishes.
 
 ### New Features
 
+- feat: add an option to make the custom marker editable (#117) - thanks to @ryamaguchi0220
+- feat: added firing dragover event (#107) - thanks to @apwidejulien
+- feat: return the id of the custom time if clickedtemplates (#96) - thanks to @ryamaguchi0220
```
Comment (ryamaguchi0220) [inline HISTORY.md]:
```suggestion
- feat: return the id of the custom time if clicked (#96) - thanks to @ryamaguchi0220
```

**Comment by yotamberk** [comment]
> Will #132 not be released?

It will. I guess I missed it in the commits.

**Pull request merged**